react/src/components/atoms/forms/Form/Form.stories.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { withKnobs } from '@storybook/addon-knobs/react';
+
+import Form, { FormProvider } from './index';
+import InputNumber from '../InputNumber';
+import InputNumberOptions from '../InputNumber/InputNumber.knobs.options';
+import InputSliderOptions from '../InputSlider/InputSlider.knobs.options';
+import InputSlider from '../InputSlider';
+
+storiesOf('atoms/forms', module).addDecorator(withKnobs)
+  .add('Form', withInfo(`<div></div>`)(() => {
+    const inputTextOptionsWithKnobs = Object.assign(...Object.entries(InputNumberOptions).map(([k, v]) => (
+      { [k]: v() })));
+    const inputSliderOptionsWithKnobs = Object.assign(...Object.entries(InputSliderOptions).map(([k, v]) => (
+      { [k]: v() })));
+    const ticks = [];
```

Comment:
can we update these values so that they are relevant to the domain passed eg: 
```suggestion
    const ticks = [{ 0: '0%', 60: 'Minimum requirement', 100: '100%' }];
```